src/main/scala/xiangshan/frontend/IFU.scala

```diff
@@ -820,6 +826,7 @@ class NewIFU(implicit p: Parameters) extends XSModule
   io.toIbuffer.bits.ftqOffset.zipWithIndex.map{case(a, i) => a.bits := i.U; a.valid := checkerOutStage1.fixedTaken(i) && !f3_req_is_mmio}
   io.toIbuffer.bits.foldpc      := f3_foldpc
   io.toIbuffer.bits.exceptionType := ExceptionType.merge(f3_exception_vec, f3_crossPage_exception_vec)
+  io.toIbuffer.bits.isForVS     := f3_isForVS_vec
```

Comment:
> 3. If I understand correctly, this `isForVS` flag is only used in case of guest page fault? Maybe we can store it in GPAMem together with `gpaddr`, instead of iBuffer, to avoid increasing iBuffer area?

Yes，it is only used when gpf happens